jomar-restrepo/tarea1/styles.css

```
@@ -0,0 +1,104 @@
+.fondo {
+  background-image: url("/jomar-restrepo/tarea1/img/fondop1.jpg");
```

@Leviathan9020 , you should use relative paths

```suggestion
  background-image: url("../img/fondop1.jpg");
```